.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,28 @@
+name: Benchmark
+on:
+  push:
+    branches:
+      - main
+permissions:
+  contents: write
+  packages: write
```

Comment:
I think these permissions aren't needed right?

Reply:
Maybe not, I'll remove them and check. I just copied what was in the build action